src/services/projects.js

```diff
@@ -665,15 +656,48 @@ module.exports = class ProjectsHelper {
 				await reviewsQueries.bulkCreate(reviewsData)
 			}
 
+			//update the reviews and resource status
+			let resourceStatus = common.RESOURCE_STATUS_SUBMITTED
```

Comment:
@priyanka-TL In which scenario do we have to run this code? Updating reviews when calling details?

Reply:
@aks30  If the creator is submitting the resource for the second time, the resource status should be set to "IN_REVIEW" instead of "SUBMITTED." Additionally, we need to update the status in the reviews table to "CHANGES_UPDATED" for any reviews that were previously marked as "REQUESTED_FOR_CHANGES."